### PR TITLE
Older version of content + pull from staging

### PR DESCRIPTION
**Pull request by snacksncode**

### CODE REVIEW

File: pages/patient/party.jsx [hunk 1]
```diff
@@ -44,24 +42,12 @@ const PartyPage = () => (
...
-      <img
-        src="/images/patient/transport/3.jpg"
-        alt="Zdjęcie nie załadowało się"
-        className={`${CLASS_NAME}__images-container__image`}
-      />
-    </div>
-  </Layout>
-);
+        <div className="page__images-container">
+            <img src="/images/patient/transport/1.jpg" alt="Zdjęcie nie załadowało się" className="page__images-container__image"/>
```
Comment (Darex1991) [inline pages/patient/party.jsx:46]: `page__image`

File: pages/patient/transport.jsx [hunk 2]
```diff
@@ -56,28 +56,11 @@ const TransportPage = () => (
...
-        className={`${CLASS_NAME}__images-container__image`}
-      />
-      <img
-        src="/images/patient/transport/3.jpg"
-        alt="Zdjęcie nie załadowało się"
-        className={`${CLASS_NAME}__images-container__image`}
-      />
+    <p><small>* Czas efektywnej pracy zespołu nalicza się od momentu podstawienia ambulansu do dyspozycji zamawiającego, do zakończenia zlecenia przewozowego.</small></p>
+    <div className="page__images-container">
+        <img src="/images/patient/transport/1.jpg" alt="Zdjęcie nie załadowało się" className="page__images-container__image"/>
```
Comment (Darex1991) [inline pages/patient/transport.jsx:61]: `page__image`

File: styles/about/radaStyles.scss [hunk 3]
```diff
@@ -0,0 +1,29 @@
+@import '../variables';
+
+.rada {
+    &__wrapper {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+    }
+    &__image-container__image {
```
Comment (Darex1991) [inline styles/about/radaStyles.scss:9]: singe nesting

File: styles/globalStyles.scss [hunk 4]
```diff
@@ -38,6 +42,38 @@ body {
     width: 100%;
 }
 
+.page__title {
+    font-size: $font-size-xl;
+    font-weight: 700;
```
Comment (Darex1991) [inline styles/globalStyles.scss:47]: variables

File: styles/globalStyles.scss [hunk 5]
```diff
@@ -38,6 +42,38 @@ body {
...
+        display: block;
+        height: 3px;
+        margin-top: 6px;
+        width: 48px;
+    }
+    &--small {
+        font-size: $font-size-base;
+    }
+}
+.page__images-container {
```
Comment (Darex1991) [inline styles/globalStyles.scss:60]: please create parent `.page` and nest it

File: styles/globalStyles.scss [hunk 6]
```diff
@@ -48,6 +84,15 @@ a {
             text-decoration: underline;
         }
     }
+    .page__images-container {
```
Comment (Darex1991) [inline styles/globalStyles.scss:87]: after that use `#{$this}__images-container`

File: styles/globalStyles.scss [hunk 7]
```diff
@@ -48,6 +84,15 @@ a {
...
+        flex-wrap: nowrap;
+        &__image {
```
Comment (Darex1991) [inline styles/globalStyles.scss:89]: `#{$this}__image:not(:last-child)`

File: styles/patient/nightCareStyles.scss [hunk 8]
```diff
@@ -0,0 +1,108 @@
+@import '../variables';
+
+.night-care {
+    &__card-wrapper {
+        margin-bottom: 16px;
+        &-column {
```
Comment (Darex1991) [inline styles/patient/nightCareStyles.scss:6]: please check
```
&-column + &-column {
            margin-top: 16px;
            
        }

File: styles/patient/nightCareStyles.scss [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+        & + & {
+            margin-top: 16px;
+        }
+        &-title {
+            font-size: $font-size-lg !important;
+            font-weight: 700;
+            margin: 0;
+            text-align: center;
+        }
+        .button {
```
Comment (Darex1991) [inline styles/patient/nightCareStyles.scss:31]: please move this on the end of `&__card` object

File: styles/patient/nightCareStyles.scss [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+        th {
+            background-color: darken($blue, 5);
+            color: $txt_color_light;
+            font-size: $font-size-base;
+        }
+        td {
+            font-size: $font-size-sm;
+        }
+    }
+
```
Comment (Darex1991) [inline styles/patient/nightCareStyles.scss:87]: ✂️

File: styles/patient/nightCareStyles.scss [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+}
+
+@media ( min-width: $max_width_cap ) {
+    .night-care {
+
```
Comment (Darex1991) [inline styles/patient/nightCareStyles.scss:92]: ✂️

File: styles/patient/pozStyles.scss [hunk 12]
```diff
@@ -0,0 +1,61 @@
+@import '../variables';
+
+.poz {
```
Comment (Darex1991) [inline styles/patient/pozStyles.scss:3]: what mean `poz`, name should be understandble

File: styles/patient/pozStyles.scss [hunk 13]
```diff
@@ -0,0 +1,61 @@
...
+        text-align: center;
+        &--red {
+            border-color: $red;
+            color: darken($red, 10);
+            background: lighten($red, 37.5);
+        }
+        &--blue {
+            border-color: darken($blue, 10);
+            color: darken($blue, 10);
+            background: lighten($blue, 25);
```
Comment (Darex1991) [inline styles/patient/pozStyles.scss:18]: alphabetic everywhere

File: styles/services/stationStyles.scss [hunk 14]
```diff
@@ -0,0 +1,63 @@
+@import '../variables';
+.skp {
```
Comment (Darex1991) [inline styles/services/stationStyles.scss:2]: `skp`?

File: styles/services/stationStyles.scss [hunk 15]
```diff
@@ -0,0 +1,63 @@
...
+            width: 10%;
+        }
+        tr td:nth-child(2), tr th:nth-child(2) {
+            width: 70%;
+        }
+        tr td:nth-child(3), tr th:nth-child(3),
+        tr td:nth-child(4), tr th:nth-child(4) {
+            width: 10%;
+        }
+        tr td:nth-child(4), tr th:nth-child(4),
```
Comment (Darex1991) [inline styles/services/stationStyles.scss:38]: what if there will be more child?

File: styles/services/stationStyles.scss [hunk 16]
```diff
@@ -0,0 +1,63 @@
...
+        }
+        td {
+            font-size: $font-size-sm;
+        }
+    }
+}
+
+@media (min-width: $max_width_cap ) {
+    .skp {
+        &__table {
```
Comment (Darex1991) [inline styles/services/stationStyles.scss:59]: merge this with parent

File: styles/services/stationStyles.scss (hunk 15 above)
Comment (snacksncode) [reply styles/services/stationStyles.scss:38]: Its columns, i dont think there will be more columns haha. And even if so. If width goes >100% it just uses content inside itself as width (from my knowledge)

Comment (Darex1991) [reply styles/services/stationStyles.scss:38]: II think you don't understand my question. You styled everything. Td and th the using nth-child. This is adaptive styling.

**Pull request merged**